sampleapp/templates/csr/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine
+
+COPY src/gopath/bin/mdservice /go/bin/mdservice
```

Comment:
What does the `md` in `mdservice` stand for?

Reply:
Uhh came from copy pasta, renamed